### PR TITLE
Updates to Extracts and API Explorer changes

### PR DESCRIPTION
**Pull request by RoAnnC1**
Removed numbering from API Explorer
Extracts moved opening para, added text from Integrate, minor updates to spacing, caps, etc.

### CODE REVIEW

File: src/api-reference/common/extracts/v1.extracts.markdown [hunk 1]
```diff
@@ -63,7 +69,7 @@ None
 
 #### Payload
 
-Name | Type | Format | Description
+Name|Type|Format|Description
 -----|------|--------|------------
```
Comment (charlieconcur) [inline src/api-reference/common/extracts/v1.extracts.markdown]:
```suggestion
---|---|---|---
```

This is the most minimal amount of Markdown to specify a table + columns. If we are going to fix whitespace issues might as well get all of the tables to conform to the documentation template.

**Review by charlieconcur — CHANGES_REQUESTED** [review]

**Comment by sarraloew** [comment]
:shipit:

File: src/api-reference/common/extracts/v1.extracts.markdown (hunk 1 above)
Comment (RoAnnC1) [reply src/api-reference/common/extracts/v1.extracts.markdown]: Fixed

**Review by charlieconcur — APPROVED** [review]

**Comment by charlieconcur** [comment]
:shipit:

**Pull request merged**